examples/run_CPMG.py

```diff
@@ -149,4 +141,15 @@
 print("\n*** FILE SAVED IN TARGET DIRECTORY ***\n")
 print(("Name of saved data", data.name()))
 config_dict.write()
-fl.show()
+# {{{ Image raw data
+with figlist_var() as fl:
+    fl.next("Raw - time")
+    fl.image(data.mean("nScans"))
+    data.reorder("t2", first=False)
+    for_plot = data.C
+    for_plot.ft("t2", shift=True)
+    for_plot.ft(["ph_overall", "ph_diff"], unitary=True)
+    fl.next("FTed data")
+    fl.image(for_plot.mean("nScans"))
+    fl.show()
```

Comment:
If you use a with block, you do not need a show at the end

Reply:
cool wasn't sure!